	/**
	 * Test fix for bug 65180: Spurious "Javadoc: xxx cannot be resolved or is not a field" error with inner classes
	 * @see <a href="http://bugs.eclipse.org/bugs/show_bug.cgi?id=65180">65180</a>
	 */
	public void testBug65180() {
		runNegativeTest(
			new String[] {
				"Test.java",
				"public class Test {\n" + 
					"	public class Inner {\n" + 
					"		/**\n" + 
					"		 * Does something.\n" + 
					"		 * \n" + 
					"		 * @see #testFunc\n" + 
					"		 */\n" + 
					"		public void innerFunc() {\n" + 
					"			testFunc();\n" + 
					"		}\n" + 
					"	}\n" + 
					"	\n" + 
					"	public void testFunc() {}\n" + 
					"}\n" + 
					"\n"
			},
			"----------\n" + 
				"1. ERROR in Test.java (at line 6)\r\n" + 
				"	* @see #testFunc\r\n" + 
				"	        ^^^^^^^^\n" + 
				"Javadoc: testFunc cannot be resolved or is not a field\n" + 
				"----------\n"
		);
	}

